src/encoder.rs

```diff
@@ -2934,21 +2934,19 @@ fn encode_partition_topdown<T: Pixel, W: Writer>(
     };
 
   let mut rdo_output =
-    block_output.clone().unwrap_or(PartitionGroupParameters {
+    block_output.clone().unwrap_or_else(|| PartitionGroupParameters {
       part_type: PartitionType::PARTITION_INVALID,
       rd_cost: std::f64::MAX,
       part_modes: ArrayVec::new(),
-    });
+    }
+  };
```

Comment:
Oops, my suggestion didn't include reverting this hunk.
```suggestion
  });
```